Packs/CommonScripts/Scripts/FormatURL/FormatURL_test.py

```diff
@@ -119,6 +119,17 @@
 
 FORMAT_URL_TEST_DATA = NOT_FORMAT_TO_FORMAT + REDIRECT_TEST_CASES + FORMAT_URL_ADDITIONAL_TEST_CASES
 
+SINGLE_LETTER_TLD = [
+    ('www.test.com', False),
+    ('a./b', True),
+    ('goog./', True),
```

Comment:
@yaakovi the 127 test you added will return false because it will not parse it as a url. will not find the '/' seperator from the TLD so it will ignore it and return false. is that a mistake? what did you try to test in this case?